tasks/main.yml

```
@@ -102,6 +102,22 @@
       {%   endif %}
       {% endfor %}
 
+- name: Configure additional files
+  copy:
+    content: "{{ file['content'] }}"
+    dest: /etc/postfix/{{ file['name'] }}
+    owner: root
+    group: root
+    mode: '0644'
```

```suggestion
    mode: '0644'
```
If the file contains secrets, those secrets will be world readable

---

Done!